### ISSUE TITLE
designing and creating a footer for the website with Figma + React [GSSoC'23]

### ISSUE THREAD

**Issue by 1Shubham7** [amanjaiman1/Product_3D#178]
### Is your feature request related to a problem? Please describe.

There is no footer present on the website. A good footer designed on Figma and then coded would make the website look more professional and better looking.

### Describe the solution you'd like

I proposal is to allow me to create a figma design and then code a footer for the website.

Sample footer design :
![sample footer design](https://github.com/amanjaiman1/Product_3D/assets/116020663/f7629d1c-fc83-4ed2-b8c9-15d676a6022a)

### Additional context

Please Assign me this issue under GSSoC'23

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 1Shubham7** [comment]
@amanjaiman1 , Yes I know. I thought I could design  a better looking and interesting footer. Should I pursue or close the issue?

**Comment by amanjaiman1** [comment]
Like any design you have , by which i assign you the work

**Comment by 1Shubham7** [comment]
@amanjaiman1 I didn't understand you. can you please repeat.

**Comment by 1Shubham7** [comment]
If you are asking for design. I have sent you a sample in the issue itself.

**Comment by amanjaiman1** [comment]
Bro choose any other problem or feature you can add.

**Comment by 1Shubham7** [comment]
ok. Should I close this one?

**Comment by amanjaiman1** [comment]
Yeah
